diff -urNp -x '*.orig' dev86-0.16.21.org/elksemu/Makefile dev86-0.16.21/elksemu/Makefile
--- dev86-0.16.21.org/elksemu/Makefile	2013-10-29 16:44:51.000000000 +0100
+++ dev86-0.16.21/elksemu/Makefile	2024-06-09 15:11:41.740995836 +0200
@@ -42,7 +42,7 @@ dummy:
 # to redirect.
 install: elksemu
 	install -d $(DIST)/lib
-	install -s -o root -g root -m 4555 elksemu $(DIST)/lib/elksemu
+	install -m 755 elksemu $(DIST)/lib/elksemu
 
 clean realclean:
 	rm -f $(OBJ) binfmt_elks.o elksemu call_tab.v defn_tab.v efile.h
