diff -Nur dev86-0.16.0.orig/libbsd/Make.defs dev86-0.16.0/libbsd/Make.defs
--- dev86-0.16.0.orig/libbsd/Make.defs	Tue Apr 23 22:46:59 1996
+++ dev86-0.16.0/libbsd/Make.defs	Sun Nov 11 19:26:59 2001
@@ -53,7 +53,7 @@
 CC=gcc $(ARCH)
 # ARCH=-b i486-linuxaout
 LKFLAGS=-static -N
-CCFLAGS=-O6 -fomit-frame-pointer -I- -I$(TOP)/include -I. -fno-builtin
+CCFLAGS=$(OPT) -fomit-frame-pointer -I- -I$(TOP)/include -I. -fno-builtin
 WALL=	-ansi -pedantic -Wwrite-strings -Wpointer-arith -Wcast-qual \
 	-Wcast-align -Wtraditional -Wstrict-prototypes -Wmissing-prototypes \
 	-Wnested-externs -Winline -Wshadow
--- dev86-0.16.18/Makefile.orig	2010-10-23 17:31:44.000000000 +0200
+++ dev86-0.16.18/Makefile	2012-04-05 16:04:39.032535007 +0200
@@ -18,7 +18,7 @@
 INCLDIR= $(PREFIX)/lib/bcc
 ASLDDIR= $(BINDIR)
 MANDIR=	 $(PREFIX)/man
-CFLAGS=  -O
+CFLAGS=  $(OPT)
 IFDEFNAME= ifdef
 
 # Some makes take the last of a list as the default ...
--- dev86-0.16.8/makefile.in.orig	Tue Aug 27 21:57:45 2002
+++ dev86-0.16.8/makefile.in	Tue Aug 27 22:00:41 2002
@@ -40,7 +40,7 @@
 WALL  =-Wall -Wstrict-prototypes
 
 CC	=%CC%
-CFLAGS	=$(GCCFLAG) $(WALL) -O2 -g
+CFLAGS	=$(GCCFLAG) $(WALL) $(OPT)
 #endif
 
 #ifndef GNUMAKE
--- dev86-0.16.8/dis88/Makefile.orig	Fri Jul 31 15:01:40 1998
+++ dev86-0.16.8/dis88/Makefile	Tue Aug 27 22:38:03 2002
@@ -25,7 +25,7 @@
 # be necessary to alter the formats of the tables.
 
 #CC=bcc
-CFLAGS=-O
+CFLAGS=$(OPT)
 LDFLAGS=
 PREFIX=/usr
 
